src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -28,62 +32,66 @@ void setUp() {
     }
 
     @Test
-    void register_noExistUserLogin_Ok() {
-        if (Storage.people.size() != 0) {
-            assertEquals(user, registrationService.register(user));
-        }
+    void register_noExistUserLogin_ok() {
+        assertEquals(user, registrationService.register(user));
     }
 
     @Test
-    void register_containsUserLogin_NotOk() {
+    void register_successfulRegistration_ok() {
+        registrationService.register(user);
+        assertEquals(user, storageDao.get(user.getLogin()));
+    }

```

Comment:
looks like we can combine these methods and just write two assertEquals inside one test